tests/integration/createAndEndActivities.intTest.ts

```diff
@@ -108,13 +108,13 @@ describe('PUT /activities/:id/end', () => {
   const request = supertest(`http://localhost:${config.serverless.port}`);
 
   context('when a non-existing activity is ended', () => {
-    it('should respond with HTTP 404', () => {
+    it('should respond with HTTP 204', () => {
       return request
         .put(`/activities/bad_id/end`)
         .send({})
         .expect('access-control-allow-origin', '*')
         .expect('access-control-allow-credentials', 'true')
-        .expect(404);
+        .expect('204');
```

Comment:
Is there a reason this one is not an integer like all the others?
'204' instead of 204.

Reply:
This has now been amended so that it is an integer value.